modin/pandas/io.py

```diff
@@ -504,6 +826,22 @@ def read_sql(
 ):  # noqa: PR01, RT01, D200
     """
     Read SQL query or database table into a DataFrame.
+
+    Parameters Support Status:
+
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | Parameters              | PandasOnRay     | PandasOnDask   | OmniSci        | Notes                            |
+    +=========================+=================+================+================+==================================+
+    | All parameters          |                 |                | Harmful        | OmniSci execution is not         |
+    |                         |                 |                | Non-lazy       | supported                        |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | con                     |                 |                | Harmful        | **Ray/Dask**:                    |
+    |                         |                 |                | Non-lazy       | Please use                       |
```

Comment:
Why are there empty cells for Ray and Dask?

Reply:
These cells are empty because for Ray/Dask these parameters are supported and we don't have any comments/notes/flags for them (by default parameter is supported if table doesn't contain any notes for this parameter). `con` parameter was added here to add flags for OmniSci execution and notes for Ray/Dask engines.

Reply:
Got it, thanks